src/qililab/platform/platform.py

```diff
@@ -330,6 +332,15 @@ def __init__(self, runcard: Runcard):
         self.experiment_results_path_format: str = "{date}/{time}/{label}.h5"
         """Format of the experiment results path."""
 
+        self.crosstalk: CrosstalkMatrix | None = None
```

Comment:
This is the source of truth! No need to re-store crosstalk in `structured_program` or somewhere else.

If users want to assign a new crosstalke **during** executing, they will use the new `SetCrosstalk` operation.